pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableUtils.java

```diff
@@ -107,6 +108,34 @@ public static DataTable buildEmptyDataTable(QueryContext queryContext)
     }
   }
 
+  public static DataTable constructMetadataOnlyDataTable(DataTable origDataTable) {
```

Comment:
(minor) for consistency
```suggestion
  public static DataTable buildMetadataOnlyDataTable(DataTable origDataTable) {
```

We should also have one to `buildDataOnlyDataTable`